querybook/webapp/components/DataDocStatementExecution/DataDocStatementExecution.tsx

```diff
@@ -148,13 +151,36 @@ export const DataDocStatementExecution: React.FC<IProps> = ({
                     />
                 </div>
             );
+            const estimatedTimeDOM = (() => {
+                if (percentComplete === 0 || percentComplete === 100) {
+                    return null;
+                }
+
+                const secondsPassedSinceStart =
+                    new Date().getTime() / 1000 - createdAt;
+                const secondsRemainToComplete =
+                    secondsPassedSinceStart / (percentComplete / 100) -
+                    secondsPassedSinceStart;
+
+                return (
+                    <div>
+                        Estimated to be completed in{' '}
+                        {formatDuration(
+                            moment.duration(secondsRemainToComplete, 'seconds')
+                        )}
+                        .
+                    </div>
```

Comment:
would prefer to format the string first to avoid the weird `{' '}`, e.g.
```
message = `Estimated to be completed in ${formatDuration(moment.duration(secondsRemainToComplete, 'seconds'))}.`

<div>{message}</div>
```